### DISCUSSION TITLE
Why not provide the visualization as a package?

### DISCUSSION THREAD

**Discussion by dinfer** [statelyai/xstate#2307] [General]
I need to visualize the fsm, and statecharts.io may not be accessed because of firewall. I have to visualize it "offline". `@xstate/viz` is not works will. Is there any plan to publish the visualization package code?

**Comment by davidkpiano** [comment]
> Is there any plan to publish the visualization package code?

Yes there is. We're working on it now, but it won't be exactly the same as the one on statecharts.io.

**Reply by dinfer** [reply]
awesome! Where is the repo?😄 I think I can contribute if possible
